Tests/LibGfx/test-inputs/jbig2/json/bitmap-halftone-10bpp-mmr.json

```diff
@@ -28,7 +28,7 @@
                 "pattern_width": 16,
```

Comment:
Might be overkill, but you could add a CI check to ensure that re-creating the images from the files gives the same output. This would prevent these kinds of oopses (at least for nonconcurrent PRs).

Reply:
#26312